tests/integration/utils/test_poc.py

```diff
@@ -75,28 +75,43 @@ def test_drop_unknown_references(metadata, data):
 
     cleaned_data = drop_unknown_references(metadata, data)
     metadata.validate_data(cleaned_data)
+    captured = capsys.readouterr()
 
     # Assert
     pd.testing.assert_frame_equal(cleaned_data['parent'], data['parent'])
     pd.testing.assert_frame_equal(cleaned_data['child'], data['child'].iloc[:4])
     assert len(cleaned_data['child']) == 4
+    expected_output = (
+        'Success! All foreign keys have referential integrity.\n'
+        'Summary of the number of rows dropped:\n'
+        'Table Name  # Rows (Original)  # Invalid Rows  # Rows (New)\n'
+        '     child                  5               1             4\n'
+        '    parent                  5               0             5'
+    )
+    assert captured.out.strip() == expected_output
 
 
-def test_drop_unknown_references_valid_data(metadata, data):
+def test_drop_unknown_references_valid_data(metadata, data, capsys):
     """Test ``drop_unknown_references`` when data has referential integrity."""
     # Setup
     data = deepcopy(data)
     data['child'].loc[4, 'parent_id'] = 2
 
     # Run
     result = drop_unknown_references(metadata, data)
+    captured = capsys.readouterr()
 
     # Assert
     pd.testing.assert_frame_equal(result['parent'], data['parent'])
     pd.testing.assert_frame_equal(result['child'], data['child'])
+    expected_message = (
+        'Success! All foreign keys have referential integrity.\n'
+        'No rows were dropped.'
+    )
+    assert captured.out.strip() == expected_message
```

Comment:
I would rather we not have a special case message for this case. In the spec, we said we should print out for every table all the time.

Even if nothing was dropped, I think it's useful reassurance to the user that we checked all tables. Plus, it simplifies our logic.